examples/dynet-benchmark/README.md

```diff
@@ -0,0 +1,17 @@
+# dynet-benchmark (last updated Dec 13, 2017)
+
+- [rnnlm-batch](rnnlm-batch.jl): A recurrent neural network language model with mini-batched training.
+- [bilstm-tagger](bilstm-tagger.jl): A tagger that runs a bi-directional LSTM and selects a tag for each word.
+- [bilstm-tagger-withchar](bilstm-tagger-withchar.jl): Similar to bilstm-tagger, but uses characer-based embeddings for unknown words.
+- [trenn](trenn.jl): A text tagger based on tree-structured LSTMs.
+
+This directory contains examples implemented for [dynet-benchmark](https://github.com/neulab/dynet-benchmark) repo. See [DyNet technical report](https://arxiv.org/abs/1701.03980) for the architectural details of the implemented examples.
```

Comment:
Maybe say a few words here about how to use the programs, what data they use, the fact that they download their own data, they have a help option that lists all options etc.

Reply:
OK, I'll look for what I could do.